Native_UWP_V2/MainPage.xaml.cs

```diff
@@ -1,55 +1,49 @@
-﻿using Microsoft.Identity.Client;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Microsoft.Graph;
+using Microsoft.Identity.Client;
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
-using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Threading.Tasks;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
 
 // The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
 
-namespace active_directory_dotnet_native_uwp_v2
+namespace Native_UWP_V2
 {
     /// <summary>
     /// An empty page that can be used on its own or navigated to within a Frame.
     /// </summary>
     public sealed partial class MainPage : Page
     {
-        //Set the API Endpoint to Graph 'me' endpoint
-        readonly string graphAPIEndpoint = "https://graph.microsoft.com/v1.0/me";
-
         //Set the scope for API call to user.read
-        string[] scopes = new string[] { "user.read" };
+        private string[] scopes = new string[] { "user.read" };
 
-        // Below are the clientId (Application Id) of your app registration and the tenant information. 
+        // Below are the clientId (Application Id) of your app registration and the tenant information.
         // You have to replace:
         // - the content of ClientID with the Application Id for your app registration
-        // - Te content of Tenant by the information about the accounts allowed to sign-in in your application:
-        //   - For Work or School account in your org, use your tenant ID, or domain
-        //   - for any Work or School accounts, use organizations
-        //   - for any Work or School accounts, or Microsoft personal account, use common
-        //   - for Microsoft Personal account, use consumers
-        private const string ClientId = "4a1aa1d5-c567-49d0-ad0b-cd957a47f842";        
+        private const string ClientId = "[Application Id pasted from the application registration portal]";
```

Comment:
I completely disagree with not having pre-registererd applications for public client applications.
Please start an email thread with Henrik, Saeed, and the Symbol team.
cc: @henrik-me @jennyf19 @SaeedAkhter-MSFT

Reply:
I think I replied to the email, but agree that pre-registered samples that work 'out of the box' seem to be preferable.

Reply:
which email ?

---
In reply to: [430115259](https://github.com/Azure-Samples/active-directory-dotnet-native-uwp-v2/pull/38#discussion_r430115259) [](ancestors = 430115259)

Reply:
@Shama-K : I still disagree :)

Reply:
Updated the sample to be pre-configured.
Thanks :)